packages/ag-charts-community/src/chart/mapping/prepareSeries.test.ts

```diff
@@ -144,258 +186,560 @@ describe('transform series options', () => {
+                area: areas,
+                column: columns,
+                line: lines,
+                nightingale: nightingales,
+                'range-column': rangeColumns,
+            };
+
+            Object.entries(seriesTypes).forEach(([seriesType, { stackable, groupable, stackedByDefault }]) => {
+                if (stackable) {
+                    addStackableSeriesType(seriesType);
+                }
+                if (groupable) {
+                    addGroupableSeriesType(seriesType);
+                }
+                if (stackedByDefault) {
+                    addStackedByDefaultSeriesType(seriesType);
+                }
+            });
+
+            beforeEach(() => {
+                console.warn = jest.fn();
+            });
+
+            it.each(Object.keys(seriesTypes))(
+                `should handle options stacked property 'true' appropriately`,
+                (seriesType) => {
+                    const sType = seriesType as keyof typeof seriesTypes;
+                    const sOptions = stackingSeriesOptions[sType];
+                    const result: typeof sOptions = processSeriesOptions(
+                        {},
+                        sOptions.map((s) => ({ ...s, stacked: true }))
+                    );
+                    const groupable = seriesTypes[sType].groupable;
+                    const stackable = seriesTypes[sType].stackable;
+                    const stacked = stackable;
+                    const grouped = !stacked && groupable;
+                    result.forEach((s) => {
+                        if (stackable) {
+                            expect(s.stacked).toBe(stacked);
+                            expect(s.grouped).toBe(grouped);
+                        } else {
+                            expect(console.warn).toBeCalled();
+                        }
+                    });
+                }
+            );
+
+            it.each(Object.keys(seriesTypes))(
+                `should handle options stacked property 'false' appropriately`,
+                (seriesType) => {
+                    const sType = seriesType as keyof typeof seriesTypes;
+                    const sOptions = stackingSeriesOptions[sType];
+                    const result: typeof sOptions = processSeriesOptions(
+                        {},
+                        sOptions.map((s) => ({ ...s, stacked: false }))
+                    );
+                    const groupable = seriesTypes[sType].groupable;
+                    const stackable = seriesTypes[sType].stackable;
+                    const stacked = false;
+                    const grouped = !stacked && groupable;
+                    result.forEach((s) => {
+                        if (stackable) {
+                            expect(s.stacked).toBe(stacked);
+                            expect(s.grouped).toBe(grouped);
+                        }
```

Comment:
Should we make an assertion in the negative case as a sanity check?

Reply:
So at the moment we don't do a waring if the options set `stacked: false` and the series is not stackable, the warning only appears if it's been set to true and the series isn't stackable